.github/workflows/license_scan.yml

```diff
@@ -0,0 +1,33 @@
+name: Trivy License Scans
+
+on:
+  pull_request:
+    branches: [ "main" ]
+
+
+jobs:
+  build:
+    name: build
+    runs-on: ubuntu-20.04
+    timeout-minutes: 30
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Use node.js
+        uses: actions/setup-node@v3
+      - run: npm i
```

Comment:
Use `npm ci` because `npm i` might upgrade packages and change `package-lock.json`.